### PR TITLE
[WIP] clean and lint code

### PR DESCRIPTION
**Pull request by romainmartinez**
Just removing useless stuff and lint code with [black](https://github.com/psf/black)--which is now widely used in the community. To use black, `pip install black` and `black pyomeca/` before pushing to GitHub. Or Pycharm integration is described in repo if needed.

Related to #69

### CODE REVIEW

**Comment by romainmartinez** [comment]
Ready to be merged

File: examples/signal_processing.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 MARKERS_ANALOGS_C3D = DATA_FOLDER / "markers_analogs.c3d"
 
 # read an emg from a c3d file
-a = Analogs3d.from_c3d(MARKERS_ANALOGS_C3D, names=["EMG1"])
+a = Analogs3d.from_c3d(MARKERS_ANALOGS_C3D, names=["Delt_ant.EMG1"])
```
Comment (pariterre) [inline examples/signal_processing.py]: I think we will have a duplicate on this?

Comment (pariterre) [reply examples/signal_processing.py]: Or is that my fix?

Comment (pariterre) [reply examples/signal_processing.py]: Ok, that is definitely mine!
PR approved!

**Pull request merged**